File: MyTournamentApp/app/src/main/java/com/example/ourtournament/Objetos/Noticia.java
package com.example.ourtournament.Objetos;

import android.widget.ImageView;

import java.sql.Date;

public class Noticia {
    public int IDNoticia;
    public String Torneo;
    public String Titulo;
    public String Descripcion;
    public Boolean Destacada;
    public int Foto;
    public Date Fecha;
    public Noticia(int idNoticia, String torneo, String titulo, String descripcion, Boolean destacada, int foto, Date fecha)
    {
        IDNoticia = idNoticia;
        Torneo = torneo;
        Titulo = titulo;
        Descripcion = descripcion;
        Destacada = destacada;
        Foto = foto;
        Fecha = fecha;
    }
    public Noticia() {
    }

}

File: MyTournamentAPI/OurTournamentAPI/Models/Noticia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class Noticia
    {
        private int IDnoticia;
        private int IDtorneo;
        private string titulo;
        private string descripcion;
        private bool destacada;
        private int foto;
        private DateTime fecha;

        public int IDNoticia { get => IDnoticia; set => IDnoticia = value; }
        public int IDTorneo { get => IDtorneo; set => IDtorneo = value; }
        public string Titulo { get => titulo; set => titulo = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
        public bool Destacada { get => destacada; set => destacada = value; }
        public int Foto { get => foto; set => foto = value; }
        public DateTime Fecha { get => fecha; set => fecha = value; }

        public Noticia(int idNoticia, int Idtorneo, string Titulo, string Descripcion, bool Destacada, int Foto, DateTime Fecha)
        {
            IDnoticia = idNoticia;
            IDtorneo = Idtorneo;
            titulo = Titulo;
            descripcion = Descripcion;
            destacada = Destacada;
            foto = Foto;
            fecha = Fecha;
        }
        public Noticia()
        {
        }
    }
}